### ISSUE TITLE
Disable the minify function on dev environment

### ISSUE THREAD

**Issue by PheysX** [FriendsOfShopware/FroshPlatformHtmlMinify#8]
### Please describe the feature you would like to see implemented.

Related to #4, i had a similar problem on my dev system. 

When i use the symfony dump feature in twig files, the dump output is completely destroyed.

![image](https://user-images.githubusercontent.com/28557712/136209846-c06a97cb-2dd1-417f-8707-03e8cebb1a91.png)

First of all, i know, the purpose of this plugin is the production and not the development usage. But i dont want to disable/uninstall the plugin. :D 

So the question is: Would it not be a good idea to disable the minify function automatically on dev environment?

**Comment by PheysX** [comment]
Oh wait, is this an exclusion for my case? https://github.com/FriendsOfShopware/FroshPlatformHtmlMinify/commit/a3ff77ff67920c7fa07a76c11e97ea932b357130

**Comment by tinect** [comment]
> Oh wait, is this an exclusion for my case? [a3ff77f](https://github.com/FriendsOfShopware/FroshPlatformHtmlMinify/commit/a3ff77ff67920c7fa07a76c11e97ea932b357130)

Yes, this would be a hotfix. But you're right! Disabling the function on dev env would be the best.

**Comment by tinect** [comment]
You are welcome to make a PR. Otherwise, I'll check this next days :-)
